provisions/roles/cleanup/openshift/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Stop and remove mail-server container
+  sudo: yes
+  docker_contianer:
+    name: mail-server
+    state: absent
+    force_kill: yes
+
+- name: Stop and remove OpenShift origin container
+  sudo: yes
+  docker_contianer:
+    name: origin
+    state: absent
+    force_kill: yes
```

Comment:
We have to add registry and router also here and pod deployer will also not be in exited state. please `docker rm -f $(docker ps -qa)` would help here. but I would suggest to have a setup running somewhere and write this, rather than putting thoughts on this